utils/archived-lm.json

```diff
@@ -22,6 +22,22 @@
                     "url": "https://lib.ballerina.io/",
                     "id": "swan-lake-api-docs"
                 },
+                {
```

Comment:
3.1 & 3.2 releases are missing in both json files

Reply:
Fixed.